roadmap/implementors-guide/guide.md

```diff
@@ -1043,6 +1045,24 @@ I have a basic implementation of this code on the [`ashley-test-statement-gossip
 
 ----
 
+## Gossip Protocols
+
+A lot of a relay chain's communication happens via gossiping messages. To avoid this becoming an attack vector, we want to use [Bounded Gossip Protocols](https://github.com/w3f/research/blob/master/docs/polkadot/networking/0-overview.md#bounded-gossip-protocols).
+
+### Functionality
+
+Several components make up a gossiping system:
+
+* A network implementation, that contains things like the actual connections to peers.
+* A propagation pool of messages.
+
+When the node wants to gossip a message:
+* The message is put into the propagation pool and stays there until cleaned up.
+* The message is sent out to the nodes that it is currently connected to.
```

Comment:
However, this is not accurate and does not capture all of the aspects of bounded gossip protocols. We only send out messages to peers that pass the filtration criterion for that peer. And then later on we may "unlock" those messages once we are aware of the filtration criterion for that peer changing.